### DISCUSSION TITLE
How to add extension module navigation items ?

### DISCUSSION THREAD

**Discussion by Jeet-Metapercept** [directus/directus#11576] [Archive]
I am able to register the extension, but can't find a way to add items to the navigation. How do add navigation items on the left side?

![image](https://user-images.githubusercontent.com/17977232/153478182-92a9a947-88b1-4286-8642-12247c31ae0d.png)

extension module index.js
![image](https://user-images.githubusercontent.com/17977232/153478344-f824ebc1-cddb-41cb-b9e9-8e886caa41a4.png)

**Comment by benhaynes** [comment]
CC @rijkvanzanten @erondpowell

**Comment by Jeet-Metapercept** [comment]
Searching more on the topic. I found a solution here - [1158807](https://github.com/directus/directus/discussions/7352#discussioncomment-1158807)
